packages/core/src/components/ui/ProductDetails/ProductDetailsSettings.tsx

```diff
@@ -110,13 +127,26 @@ function ProductDetailsSettings({
     <>
       {!outOfStock && (
         <section data-fs-product-details-values>
-          <ProductPrice.Component
-            data-fs-product-details-prices
-            value={price}
-            listPrice={listPrice}
-            formatter={useFormattedPrice}
-            {...ProductPrice.props}
-          />
+          <div data-fs-product-details-values-wrapper>
+            <ProductPrice.Component
+              data-fs-product-details-prices
+              value={
+                taxesConfiguration?.usePriceWithTaxes ? priceWithTaxes : price
+              }
+              listPrice={
+                taxesConfiguration?.usePriceWithTaxes
+                  ? listPriceWithTaxes
+                  : listPrice
+              }
+              formatter={useFormattedPrice}
+              {...ProductPrice.props}
+            />
+            {taxesConfiguration?.usePriceWithTaxes && (
+              <UILabel data-fs-product-details-taxes-label>
```

Comment:
should this be overridable using section override v2 feature? maybe in the future?

Reply:
Yes, We can do this. Is there documentation on how to create a new section override?